data/kahet/kahet missions.txt

```diff
@@ -303,6 +303,579 @@ mission "Ringworld Debris: Quarg"
+			label what
+			`	"The Ka'het, as you know, are part mechanical and part biological. What may be news to you, however, is their biological self is suppressed most of the time: we have been able to determine that their stellar navigation and system management is completely left to their programming," he signs, with an unusual look of aversion in his eyes.`
+			`	"Still, like most biological beings, after their nominal lifetime of hundreds of years, they can start dying of old age. When this happens, they enter a hibernation state, calling for a Builder in the hope of being rescued. Of course, no help is coming.`
+			`	"We have observed," he continues while reading from a datapad, "other Ka'het try to patch them up as best as they can after some time, presumably following their own programming to do so. It is very unusual to find the corpse of one just laying there alone. We are interested in more or less anything that cannot be extracted from the carcass of one of their kind. Memory banks, logic terminals, and in general, all parts of their mechanical brain that usually will not survive critical battle damage."`
+			`	Despite the sign language, you notice that he seems strikingly halfhearted compared to the previous times you have encountered him. As he heads toward his Gull, the Late Season Dancing, you can't help but wonder why.`
+				decline
+
+mission "Ka'het: Death in the Graveyard 3"
+	landing
+	name "Angry Ka'het"
+	description "Destroy the Telis'het in <system> to prevent it from destroying the Maeri'het on the surface of <planet>."
+	to offer
+		has "Ka'het: Death in the Graveyard 1: done"
+	on offer
+		conversation
+			`It's daytime now. Flying in formation with the Late Season Dancing, you spot the Maeri'het on the ground easily: it has spread a pair of enormous solar wings, at least thrice as long as the body of the Ka'het itself. A look at your radar reveals more, however. There is a second Ka'het standing above the downed one, classified by your IFF as a Telis.`
+			`	"It is trying to reanimate the Maeri'het," Plume chants over the intercom, "but I do not think it can do so. It has been too long since you found it for repairs to have any effect." Among the smoke clouds you distinguish pairs of tiny arms protrude from the bottom of the Telis'het, pressing all over the corpse below it. Your ship records some electric shocks move between the two, but the Maeri'het does not react.`
+			choice
+				`	"Let's wait for it to leave then."`
+				`	"Should I engage it?"`
+					goto engage
+			`	"Ah, it is not that easy. Its programming may be telling it to heal it, but the slug will grow more frustrated with each failed attempt. You see, one of the functions left under the control of the biological side is the combat. It may not be able to decide to stop trying to heal it, but it can decide to register it as a foe, destroy it, and then leave freely as if it was never there," he concludes with a cold chant.`
```

Comment:
Reducing number of "its":
```suggestion
			`	"It is not that easy. Its programming may be telling it to heal the Maeri'het, but the slug will grow more frustrated with each failed attempt. You see, one of the functions left under the control of the biological side is combat. It may not be able to decide to stop trying to heal it, but it can decide to register the Maeri'het as a foe, destroy it, and then leave freely as if it were never there," he concludes with a cold chant.`
```